NorthwindCRUD/Controllers/CustomersController.cs

```diff
@@ -76,7 +78,10 @@ public ActionResult<OrderDto[]> GetOrdersByCustomerId(string id)
         }
 
         [HttpPost]
-        [Authorize]
+        [SwaggerResponse(400, "Your inputs do not pass validation!", typeof(Errors), "text/json")]
```

Comment:
if model is not valid this will return:
```cs
return BadRequest(ModelState);
```
which will end up to something like this:
![image](https://github.com/IgniteUI/NorthwindAPI/assets/5990334/049e9648-328f-4387-beab-8b51c69ed408)

What should return this end point in this case is `Errors`. What you need to do is something like this:
```ts
                if (ModelState.IsValid == false)
                {
                    // TODO: Validate each field here
                    if (model.ContactName != "Infragistics")
                    {
                        return BadRequest(new Errors
                        {
                            CustomErrors = new ValidationError[]
                            {
                                new ValidationError
                                {
                                    DataField = nameof(model.ContactName),
                                    Message = "Contact Name is not valid",
                                    StatusCode = 401
                                },
                            },
                        });
                    }
                }

                var mappedModel = this.mapper.Map<CustomerDto, CustomerDb>(model);
                var customer = this.customerService.Create(mappedModel);
                return Ok(this.mapper.Map<CustomerDb, CustomerDto>(customer));
```